StardewArchipelago/Extensions/GameLocationExtensions.cs

```diff
@@ -139,6 +142,14 @@ public static Point GetWarpPointTarget(this GameLocation origin, Point warpPoint
                         return new Point(warpTarget.X, warpTarget.Y);
                     }
                 }
+
+                foreach (var warp in GetBuildingWarpPoints(origin, destinationName))

```

Comment:
This is the method for warp point targets, how come it checks warp points?

Reply:
My lack of full understanding how this is all structured. Should be corrected?